File: extJSTutorial/app/model/Board.js
// Board 객체를 정의하는 JS
Ext.define('extJSTutorial.model.Board',{
    extend:'Ext.data.Model',
    fields:[
        {name:'id',type:'int'},
        {name:'title',type:'string'},
        {name:'content',type:'string'},
        {name:'writerId',type:'int'},
        {name:'nickname',type:'string'},
        {name:'entryDate',type:'string'},
        {name:'modifyDate',type:'string'} 
    ]
})

File: mongo-backend/src/main/java/com/bit/mongobackend/model/Board.java
package com.bit.mongobackend.model;


import lombok.Data;

import java.util.Date;

@Data
public class Board {
    private int id;
    private String title;
    private String content;
    private int writerId;
    private String writerNickname;
    private Date entryDate;
    private Date modifyDate;
}
